locales/en.json

```diff
@@ -1809,10 +1845,14 @@
 			"label": {
 				"maxSizeFiles": "Images, videos and files",
 				"problemDescription": "Problem description:",
-				"theProblemOccursOnTheDevice": "The problem occurs on this device"
+				"theProblemOccursOnTheDevice": "The problem occurs on this device",
+				"problemAlreadyKnown": "Is the problem already known?",
+				"knownProblemLink": "Here you can see if there are currently problems with the technical systems: ",
```

Comment:
Here you can check if there are currently issues regarding the technical systems: